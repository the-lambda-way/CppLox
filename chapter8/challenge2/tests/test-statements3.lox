var a;
print a; // Error!
