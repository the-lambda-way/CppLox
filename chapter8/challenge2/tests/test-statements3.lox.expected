Variable not initialized.
[line 2]
